components/bl-pagination/README.md

```diff
@@ -60,3 +62,48 @@ The component based on external [pagination](https://mui.com/material-ui/react-p
 @bl-customComponent-pagination-button-normalSize: 32px;
 @bl-customComponent-pagination-button-largeSize: 40px;
 ```
+
+## Usage guide
+
+We will display this data table:
+
+![Data Table](./example-images/data-table.png)
+
+Add Data Grid and Pagination component to the page:
+
+![Add Data Grid and Pagination](./example-images/add-data-grid-and-pagination.png)
+
+Configure the columns to be displayed:
+
+![Configure Data Grid](./example-images/configure-data-grid-1.png)
+![Configure Data Grid](./example-images/configure-data-grid-2.png)
+
+Add property “pageSize” to “Page Data” to some mount event:
+
+![Make variable](./example-images/add-page-size-to-page-data.jpg)
+
+Now we need to calculate how many pages we have. For that, in Pagination On Mounted event we use the block for getting the count of rows in our table and then divide this count by the count of rows on one page(pageSize). This result we round up for integer number:
+
+![Calculate pages](./example-images/calculate-page-count.png)
+
+Add “pageCount” from “Page Data” to data binding for Page Count Logic:
+
+![Add pageCount](./example-images/add-page-count-to-data-binding.png)
+
+Now we load the table's data in Pagination On Page Change event and calculate page offset and set it to “Page Data”:
+
+![Load data](./example-images/get-data-from-table.jpg)
```

Comment:
Make sense to group images. We could do it in two ways.
First - example prefix.
`usage-<what happens on pickture>.jpg`

Or we could create a separate directory for every example(i'f every example contains a lot of images).

It helps us to not mix up different images and makes pictures like `result` more obvious.